src/modules/mavlink/mavlink_receiver.h

```diff
@@ -188,11 +187,17 @@ class MavlinkReceiver
 
 	void send_storage_information(int storage_id);
 
+	/**
+	 * @brief Checks for updated uORB parameters and updates the params if required.
```

Comment:
This is incorrect as it does not check. Can you either move the check in here or update the comment?
Other than that this looks good.

Reply:
Thanks for the feedback! I pushed up a correction.